models/articles.models.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../db/connection');
+const format = require('pg-format');
+
+exports.fetchArticle = (article_id) => {
+    return db.query(
+        `SELECT * FROM articles
+        WHERE article_id = $1;`, [article_id]
+    ).then(({ rows }) => {
+        if (!rows.length) {
+            return Promise.reject({ status: 404, msg: 'Article not found' })
+        }
+        return rows;
```

Comment:
Just a small change, but could you send back the single object, instead of an array with one object in it?